### PR TITLE
TFW-4096 - Modify Value

### PR DESCRIPTION
**Pull request by tVeratti**
Screen Shot 2019-07-30 at 11 38 19 PM

### CODE REVIEW

**Comment by tVeratti** [comment]
Rebased to include the commit from #1326

File: components/universalSearch/helperModal.js [hunk 1]
```diff
@@ -77,38 +92,67 @@ export class HelperModalComponent extends PureComponent {
...
+        const { activeQuery } = this.props;
+        this.onSearchQueryAppend(activeQuery.operator);
+    }
+
+    onSearchQueryAppend = (operator) => {
+        const { activeDimension, onSubmit, mode } = this.props;
+        const { selections } = this.state;
+        const { id, label } = activeDimension;
+
+        const nodes = selections.map(s => ({ ...s, displayLabel: label, id }));
```
Comment (jasonhale) [inline components/universalSearch/helperModal.js]: Yeah, this is where we'll clash.  I appended this info to each query as they were built (added to `selections`).  That way `selections` was ready to go for submit.

Let me know your opinion of which direction makes more sense.

**Comment by jasonhale** [comment]
Tested with RCI - this is working good!  Thanks!

File: components/queryNodeList/queryNode.js [hunk 2]
```diff
@@ -19,33 +25,40 @@ export default class QueryNode extends PureComponent {
...
         tooltip = dompurify.sanitize(tooltip);
 
         return (
-          <div className={`query-node ${styleVariationClassnames}`} data-tip={tooltip} data-html={true} data-for={'queryNodeTooltip'}>
-            <span className={'query-node__symbol'}>{ prependSymbol }</span>
-            { label &&
-            <span className={'query-node__dimension'}>{ label }</span>
-            }
-            <span className={'query-node__value'}>{ value }</span>
+          <button
```
[the remark this answers is not in the document]
Comment (tVeratti) [reply components/queryNodeList/queryNode.js]: Alright I'll restructure this little guy :)

**Pull request merged**